CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ project(ftxui
   VERSION 0.1
 )
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
I don't want to force users to user C++17.
What is the reason you want to add this?

Reply:
Will fix.

Doesn’t this do the same thing as target_compile_features(... PUBLIC cxx_std_17) below btw?

Reply:
I haven't seen I am already using C++17. Sorry please ignore my comment.